01/app.js

```diff
@@ -1 +1,21 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+
+
+
+const item = document.getElementsByClassName('.comments__item comments__item--newest');
+
+console.log(item);
+
+if (item) {
+    const item1 = document.querySelectorAll('span');
```

Comment:
i tutaj dopiero `const spanList = item.querySelector('[data-info]')`
i ilość to `spanList.length`